packages/api/src/index.ts

```diff
@@ -57,23 +58,38 @@ passport.use(
     {
       clientID: env.githubClientId,
       clientSecret: env.githubSecret,
+      scope: 'user:emails',
       callbackURL: `${env.appUrl}/api/auth/github/callback`,
     },
     async (accessToken: any, refreshToken: any, profile: any, done: any) => {
+      const regex = new RegExp(/^[a-zA-Z0-9]+[@][a-zA-Z0-9]+[.][e][d][u]$/g);
```

Comment:
Be sure to use descriptive variable names so others can quickly identify what it is used for. i.e. ```emailRegex``` or ```emailPattern``` 🙂